package.json

```diff
@@ -90,14 +90,15 @@
   ],
   "license": "MIT",
   "dependencies": {
+    "react-native": "0.44.3",
     "react-native-dismiss-keyboard": "1.0.0"
   },
   "peerDependencies": {
     "react": "*",
-    "react-native": "*"
```

Comment:
We don't need to pin it, I think this library should work with more than one version